src/components/RelativeRangeDatePicker/components/Presets/Presets.scss

```diff
@@ -11,7 +11,13 @@ $block: '.#{variables.$ns}relative-range-date-picker-presets';
         align-items: center;
         gap: var(--g-spacing-2);
 
+        padding: 0 var(--_--popup-content-padding);
```

Comment:
let's add css variable, something like this
```scss
#{$block} {
    --g-list-item-padding: 0 var(--_--g-date-picker-presets-padding, 0);

    margin-inline: calc(-1 * var(--_--g-date-picker-presets-padding, 0));
    
    &__tabs {
        padding-inline: var(--_--g-date-picker-presets-padding, 0);
    }
}
```

and in `PickerDialog.scss` add:

```scss
#{$block} {
    &__content {
        --_--g-date-picker-presets-padding: var(--_--popup-content-padding);

        &_mobile {
            --_--g-date-picker-presets-padding: 0;
        }
    }
}
```

These seem to be all the changes needed.

Reply:
Thank you!

but in mobile we shoudn't remove padding, i think
<img width="1097" alt="image" src="https://github.com/gravity-ui/date-components/assets/42944548/bc9bca7e-cdab-4ef3-8e2b-58ebfa4f8956">
VS
<img width="1099" alt="image" src="https://github.com/gravity-ui/date-components/assets/42944548/bd94793e-7f16-4e89-b1cf-a94e222ea4cf">

Reply:
Yes, I forgot to change the code in my comment to

```scss
    &__presets {
        margin-inline: calc(-1 * var(--_--g-date-picker-presets-padding));
    }
```

with it the mobile view will be the same as before.